src/send-messages.ts

```diff
@@ -271,7 +271,7 @@ async function submitMessageforUser(
   subscription_key: string
 ) {
   let message = {
-    content: GetMessageContent(messageContent),
+    content: getMessageContent(messageContent),
```

Comment:
If getMessageContent returns `undefined` you are de-facto posting an invalid message.